hub/constraints/instruction-handling/acc.lisp

```diff
@@ -23,21 +23,42 @@
-                                                         [ stack/DEC_FLAG 2 ]
-                                                         [ stack/DEC_FLAG 3 ]
-                                                         [ stack/DEC_FLAG 4 ])) ;; ""
+(defconst
+  ROW_OFFSET___ACC_FAMILY___CONTEXT_ROW            1
+  ROW_OFFSET___ACC_FAMILY___ACCOUNT_READING_ROW    2
+
+  ROW_OFFSET___ACC_FAMILY___ACCOUNT_DOING_ROW      1
+  ROW_OFFSET___ACC_FAMILY___ACCOUNT_UNDOING_ROW    2
+  )
+
+
+(defun (account-instruction---is-BALANCE)               [ stack/DEC_FLAG 1 ] )
+(defun (account-instruction---is-EXTCODESIZE)           [ stack/DEC_FLAG 2 ] )
+(defun (account-instruction---is-EXTCODEHASH)           [ stack/DEC_FLAG 3 ] )
+(defun (account-instruction---is-CODESIZE)              [ stack/DEC_FLAG 4 ] )
+(defun (account-instruction---raw-address-hi)           [ stack/STACK_ITEM_VALUE_HI 1 ])
+(defun (account-instruction---raw-address-lo)           [ stack/STACK_ITEM_VALUE_LO 1 ])
+(defun (account-instruction---result-hi)                [ stack/STACK_ITEM_VALUE_HI 4 ])
+(defun (account-instruction---result-lo)                [ stack/STACK_ITEM_VALUE_LO 4 ]) ;; ""
+
+(defun (account-instruction---is-SELFBALANCE)           (-    1
+                                                              (account-instruction---is-BALANCE)
+                                                              (account-instruction---is-EXTCODESIZE)
+                                                              (account-instruction---is-EXTCODEHASH)
+                                                              (account-instruction---is-CODESIZE)))
+
+(defun (account-instruction---touches-foreign-account)  (+ (account-instruction---is-BALANCE)
+                                                           (account-instruction---is-EXTCODESIZE)
+                                                           (account-instruction---is-EXTCODEHASH)))
+(defun (account-instruction---touches-current-account)  (+ (account-instruction---is-CODESIZE)
+                                                           (account-instruction---is-SELFBALANCE)))
+
+(defun (account-instruction---no-trimming)              (- stack/ACC_FLAG (account-instruction---touches-foreign-account)))
+(defun (account-instruction---account-address-hi)       (shift   context/ACCOUNT_ADDRESS_HI     ROW_OFFSET___ACC_FAMILY___CONTEXT_ROW))
+(defun (account-instruction---account-address-lo)       (shift   context/ACCOUNT_ADDRESS_LO     ROW_OFFSET___ACC_FAMILY___CONTEXT_ROW))
+(defun (account-instruction---byte-code-address-hi)     (shift   context/BYTE_CODE_ADDRESS_HI   ROW_OFFSET___ACC_FAMILY___CONTEXT_ROW))
+(defun (account-instruction---byte-code-address-lo)     (shift   context/BYTE_CODE_ADDRESS_LO   ROW_OFFSET___ACC_FAMILY___CONTEXT_ROW))
+(defun (account-instruction---foreign-address-warmth)   (shift   account/WARMTH                 ROW_OFFSET___ACC_FAMILY___ACCOUNT_DOING_ROW))
```

Comment:
These shorthands for values in `context/` or `account/` rows were missing a `shift` !